### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by alixti** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#84] [General]
Since the new update including 22H2 version of Windows 11 this fix is no longer needed as it is part of the OS already (as it should have been from the beginning).
I just wanted to say thank you for this because it really really helped me so many times and a lot of other people too.
Also kinda eased that bad/sad feeling of not having it.
_I'm gonna kinda miss the Doctor funny icon on my system tray hehe._
Again, thank you so much!

**Comment by HerMajestyDrMona** [comment]
Thank you for your kind words :)

Yes, the drag-and-drop seems to work fine in the newest version of Windows 11. I was using the BETA builds for a few months now, so I also wasn't using [Windows11DragAndDropToTaskbarFix](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix).

If somebody needs help uninstalling the fix, here's an open issue regarding this: https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/issues/81
